### PR TITLE
New cheep and author

### PR DESCRIPTION
**Pull request by cstiitu**
Added methods for new cheep and author.

Modified constraints and requirements of fields in Cheep and Author.

### CODE REVIEW

File: src/Chirp.Infrastructure/Chirp.Repositories/CheepRepository.cs
Comment (SneekySnek) [inline src/Chirp.Infrastructure/Chirp.Repositories/CheepRepository.cs:1]: Looks gooooddd!

**Review by SneekySnek — APPROVED** [review]
All good!

**Review by MarcusHVange — APPROVED** [review]
Ser fint ud

File: src/Chirp.Infrastructure/Chirp.Repositories/CheepRepository.cs
Comment (ghost) [reply src/Chirp.Infrastructure/Chirp.Repositories/CheepRepository.cs:1]: Looks even better

**Review by ghost — APPROVED** [review]
All good

**Pull request merged**